packages/circle-demo-webapp/app/routes/wallets.$id/route.tsx

```diff
@@ -1,11 +1,12 @@
 import { ActionFunctionArgs } from '@remix-run/node';
 import { Link, useLoaderData, useParams } from '@remix-run/react';
-import type { BLOCKCHAIN, WalletSet } from 'web3-circle-sdk';
+import type { WalletSet } from 'web3-circle-sdk';
```

Comment:
Should this be coming from `~/lib/types`?

Reply:
typo. fixed